presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -284,6 +284,12 @@ public AggregatedMemoryContext aggregateUserMemoryContext()
         return new InternalAggregatedMemoryContext(operatorMemoryContext.aggregateUserMemoryContext(), memoryFuture, this::updatePeakMemoryReservations, false);
     }
 
+    // caller shouldn't close this context as it's managed by the OperatorContext
```

Comment:
cmt msg:
> Introduce revocable aggregated memory context in Operator Execution Context

change to eg `Introduce revocable aggregated memory context in OperatorContext`